clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/description/DeployAppengineDescription.groovy

```diff
@@ -29,6 +29,7 @@ class DeployAppengineDescription extends AbstractAppengineCredentialsDescription
   String stack
   String freeFormDetails
   String repositoryUrl
+  String storageAccountName  // for GCS repositories only
```

Comment:
How does a Clouddriver client know of a `storageAccountName`?

Reply:
Good point. I'll add a method to the appengine controller in a followup PR.